<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileNew</class>
 <widget class="QDialog" name="ProfileNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Profile</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <widget class="PsiTextView" name="te_desc" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>32767</height>
      </size>
     </property>
     <property name="text" stdset="0">
      <string>Keep your&lt;i&gt; Profile Name&lt;/i&gt; simple. It should be a single word comprised of only letters or numbers.&lt;br&gt;
&lt;br&gt;
The&lt;i&gt; Default Action&lt;/i&gt; is what happens when you double click a contact in your list. The choices are&lt;b&gt; Message&lt;/b&gt; and&lt;b&gt; Chat&lt;/b&gt;. You can change this later from the Options menu.&lt;br&gt;
&lt;br&gt;
Check the &lt;i&gt;Enable Emoticons&lt;/i&gt; checkbox if you'd like text such as &lt;b&gt;:-)&lt;/b&gt; to be turned into graphics like &lt;icon name=&quot;psi/smile&quot;&gt;.</string>
     </property>
     <property name="readOnly" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Profile Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_name"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="bg_defAct">
       <property name="title">
        <string>Default Action</string>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>11</number>
        </property>
        <property name="topMargin">
         <number>11</number>
        </property>
        <property name="rightMargin">
         <number>11</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="QRadioButton" name="rb_message">
          <property name="text">
           <string>&amp;Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_chat">
          <property name="text">
           <string>&amp;Chat</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ck_useEmoticons">
       <property name="text">
        <string>Enable Emoticons</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="Line1">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="spacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="IconButton" name="pb_close" native="true">
         <property name="text" stdset="0">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="IconButton" name="pb_create" native="true">
         <property name="text" stdset="0">
          <string>&amp;Create</string>
         </property>
         <property name="shortcut" stdset="0">
          <string>Alt+C</string>
         </property>
         <property name="autoDefault" stdset="0">
          <bool>false</bool>
         </property>
         <property name="default" stdset="0">
          <bool>true</bool>
         </property>
         <property name="psiIconName" stdset="0">
          <string>psi/addContact</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PsiTextView</class>
   <extends>QWidget</extends>
   <header>psitextview.h</header>
  </customwidget>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>iconbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_name</tabstop>
  <tabstop>rb_message</tabstop>
  <tabstop>rb_chat</tabstop>
  <tabstop>ck_useEmoticons</tabstop>
  <tabstop>te_desc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
